jobbergateapi2/apps/job_submissions/schemas.py

```diff
@@ -19,12 +19,7 @@ class JobSubmissionRequest(BaseModel):
 
     class Config:
         orm_mode = True
-        schema_extra = {
-            "example": {
-                "job_submission_name": "name",
-                "job_script_id": 1,
-            }
-        }
+        schema_extra = {"example": {"job_submission_name": "name", "job_script_id": 1}}
```

Comment:
Why do we have this `schema_extra`? A comment here about what this does would be helpful.

Reply:
I had to look this up myself. This just adds an example in the swagger pages. I added a comment with the link.